src/main/java/com/twilio/oai/api/PythonApiResourceBuilder.java

```diff
@@ -39,6 +39,8 @@ public ApiResourceBuilder updateOperations(final Resolver<CodegenParameter> code
             for (CodegenParameter cp : co.allParams) {
                 if (cp.paramName.equals("_from")) {
                     cp.paramName = "from_";
+                } else if (cp.paramName.equals("a2_p_profile_bundle_sid")) {
+                    cp.paramName = "a2p_profile_bundle_sid";
```

Comment:
Eep, don't like adding more of these exceptional cases. Checking if we can use the `StringHelper.toSnakeCase()` somewhere since it handles these cases ...